sklearn/tests/test_common.py

```diff
@@ -222,7 +222,7 @@ def test_transformer_n_iter():
             else:
                 yield check_transformer_n_iter, name, estimator
 
-
+@ignore_warnings(category=DeprecationWarning)
```

Comment:
The RandomizedPCA class is also deprecated, but hard coding each class isn't best practice either... How would you recommend going about it - create a list of classes?